Jenkinsfile

```diff
@@ -14,6 +14,7 @@ bparams.snykContainer = 'python:3.6-buster'
 bparams.snykRegistry = ''
 bparams.snykType = 'PYTHON'
 bparams.nexusiqtype = 'PREQUIREMENT'
+releaseBuild = 'Yes'
```

Comment:
Always Yes - should that be a build parameter?

Reply:
Currently for testing I put as Yes. We will put this parameter as 'No' and only at the time of release, we will put as 'Yes', so that it might be executed only at the release time of package